### PR TITLE
Add RGB10_A2 texture formats

### PR DESCRIPTION
**Pull request by MoochMcGee**
1-2-Switch uses these at least.

### CODE REVIEW

**Comment by Hexagon12** [comment]
Please fix the conflicts

**Comment by bunnei** [comment]
```bunnei - Today at 9:03 AM
mooch2: Couple things on your pr: 1. can you fix merge conflicts? and 2. did you test it? (i.e. use something like RenderDoc to verify textures are actually uploading properly)```

**Comment by Subv** [comment]
Please add the format before the compressed formats instead of after.

Also, add it to decoders.cpp

File: src/video_core/textures/texture.h [hunk 1]
```diff
@@ -19,6 +19,7 @@ enum class TextureFormat : u32 {
     DXT1 = 0x24,
     DXT23 = 0x25,
     DXT45 = 0x26,
+    RGB10_A2 = 0xD1,
```
Comment (bunnei) [inline src/video_core/textures/texture.h]: this is wrong

**Comment by bunnei** [comment]
Superseded by #378

**Pull request closed without merging**